### PR TITLE
background red everywhere but top story and hover 30% crimson

### PR DESCRIPTION
**Pull request by florianherrengt**
image
image
image
image

### CODE REVIEW

File: src/scss/themes/_live.scss [hunk 1]
```diff
@@ -1,30 +1,42 @@
...
-        // create white text red background
-        @include oTeaserInverse;
-        background: oColorsGetPaletteColor('crimson');
-        .o-teaser__content * {
-            color: white;
-            &:hover {
-                color: white;
-            }
+    .o-teaser__tag:hover {
+        color: oColorsMix(oColorsGetPaletteColor('crimson'), oColorsGetPaletteColor('white'), 30) !important;
```
Comment (florianherrengt) [inline src/scss/themes/_live.scss]: anyone can help me to get rid of this `important` ?

Comment (florianherrengt) [reply src/scss/themes/_live.scss]: This is what's causing the issue I think https://github.com/Financial-Times/o-teaser/blob/small-live/src/scss/themes/_standard.scss#L3-L8
I monkey patched it by targeting elements specifically but I'm still looking for a better solution https://github.com/Financial-Times/o-teaser/pull/76/commits/8065026eb99504ae72f1383dd21c3a2d2f804bd7

File: src/scss/themes/_live.scss [hunk 2]
```diff
@@ -38,4 +53,50 @@
...
+    }
+
+    .o-teaser--top-story.o-teaser--standalone.o-teaser--live .o-teaser__meta a {
+        color: oColorsGetPaletteColor('crimson');
+    }
+
+    .o-teaser--large.o-teaser--live .o-teaser__content {
+        padding: 3px 12px;
+    }
+    .o-teaser--hero, .o-teaser--large {
```
Comment (florianherrengt) [inline src/scss/themes/_live.scss]: see https://github.com/Financial-Times/o-teaser/pull/76#pullrequestreview-64943027

File: src/scss/themes/_live.scss [hunk 3]
```diff
@@ -15,19 +15,22 @@
...
-        *:hover {
+        // *:hover {
+        //     color: oColorsMix(oColorsGetPaletteColor('crimson'), oColorsGetPaletteColor('white'), 30);
+        // }
+
+        .o-teaser__heading:hover,
+        .o-teaser__heading:focus,
+        .o-teaser__tag:hover,
+        .o-teaser__tag:focus,
+        .js-teaser-heading-link:hover {
```
Comment (JakeChampion) [inline src/scss/themes/_live.scss]: How come this is `js-teaser`?

Comment (florianherrengt) [reply src/scss/themes/_live.scss]: Somehow `Heavy - large`'s meta doesn't have the class `o-teaser__tag`

image

Comment (JakeChampion) [reply src/scss/themes/_live.scss]: @florianherrengt Is that in a demo or on the live site somewhere?

Comment (florianherrengt) [reply src/scss/themes/_live.scss]: @JakeChampion Not that I'm aware of. I usually send ngrok links to people.
This is from `n-teaser`

Comment (JakeChampion) [reply src/scss/themes/_live.scss]: The specification for origami components states that the classes should begin with `o-` - http://origami.ft.com/docs/component-spec/modules/#naming-conventions

Could we change the selector from `.js-teaser` to `.o-teaser` and ask Next FT to update their html?

**Comment by florianherrengt** [comment]
Added fix for hero 

image

File: src/scss/themes/_live.scss (hunk 3 above)
Comment (florianherrengt) [reply src/scss/themes/_live.scss]: This commit should fix it https://github.com/Financial-Times/o-teaser/pull/76/commits/1742a69967d3a79eadc2a186a58101bfa83a4541 🙂

**Comment by JakeChampion** [comment]
@florianherrengt Let us know when this is ready for review 👌

**Comment by florianherrengt** [comment]
@JakeChampion This is ready for review now. IE 11 / 10 have known issues and tickets have been created. https://trello.com/c/ZTK3Cknc/610-make-living-event-homepage-teaser-better-on-ie11-and-ie10

**Comment by notlee** [comment]
It appears there is no longer a drive for these changes, so I'll close this for now.

**Pull request closed without merging**